src/js/Controllers/DisplayCapabilities.js

```diff
@@ -1,7 +1,7 @@
 let softButtonCapability = {
 	"shortPressAvailable": true,
-	"longPressAvailable": false,
-	"upDownAvailable": false,
+	"longPressAvailable": true,
+	"upDownAvailable": true,
```

Comment:
Will need to modify the values of the button capabilities in the MEDIA template as well

Reply:
Updated